scene/2d/collision_polygon_2d.cpp

```diff
@@ -98,8 +98,10 @@ void CollisionPolygon2D::_update_xform_in_parent() {
 
 	if (shape_from >= 0 && shape_to >= 0) {
 		CollisionObject2D *co = get_parent()->cast_to<CollisionObject2D>();
-		for (int i = shape_from; i <= shape_to; i++) {
-			co->set_shape_transform(i, get_transform());
+		if (co) {
```

Comment:
Auick question, why don't we use a macro here like:

```
ERR_FAIL_NULL_V(co, false);
```

Reply:
Because the situation of not having a parent is not an error. The macro would exit the function and also print a warning to the console.